surveys/templates/location_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+<div class="col col-lg-10 card">
+  <div class="card-body col-lg-9">
+    <h2 class="card-title">Delete Location</h2>
+
+     <p class="card-text">Are you sure you want to delete <b>{{object}}</b>?</p>
+     <p class="card-text">Once locations are deleted they cannot be recovered.</p>
+
+     <form method="POST" class="post-form">{% csrf_token %}
+     <button type="submit" class="save btn btn-primary">Delete</button>
```

Comment:
Could we have a `Cancel` button here, similar to the other pages?

Reply:
Good call